tests/responses/attestation/conditional-create.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "public-key",
+  "id": "KbeXjEj6HgsT_opADhRF46CXsWRR7Y8UndF03-f2sF8XRaimojYz950Mi2fN2S3DSDGyI5I3S_IZyGJbT85E6w",
+  "rawId": "KbeXjEj6HgsT_opADhRF46CXsWRR7Y8UndF03-f2sF8XRaimojYz950Mi2fN2S3DSDGyI5I3S_IZyGJbT85E6w",
+  "response": {
+    "clientDataJSON": "eyJ0eXBlIjoid2ViYXV0aG4uY3JlYXRlIiwiY2hhbGxlbmdlIjoiaUxXVlU3bTJjWV9HQjR3SVN1Q3JKUSIsIm9yaWdpbiI6Imh0dHA6Ly9sb2NhbGhvc3Q6ODA4MCIsImNyb3NzT3JpZ2luIjpmYWxzZX0",
+    "attestationObject": "o2NmbXRkbm9uZWdhdHRTdG10oGhhdXRoRGF0YVjESZYN5YgOjGh0NBcPZHZgW4_krrmihjLHmVzzuoMdl2NYAAAABAAAAAAAAAAAAAAAAAAAAAAAQCm3l4xI-h4LE_6KQA4UReOgl7FkUe2PFJ3RdN_n9rBfF0WopqI2M_edDItnzdktw0gxsiOSN0vyGchiW0_OROulAQIDJiABIVggd6q1rL84dPlHqncwa69sOrqfs300BAp1lYv9pRlqIOgiWCA8sAhHyFxsiqy4HpSRVwIG5ZMcUU3iT8evBl7DHFh-9g"
```

Comment:
This file is the same as `01-none.json` besides the `attestationObject` besides these changes:
```json
"userPresent":false
"userVerified":false
"backupEligible":true
"backupStatus":true
```